lib/cog_api/http/old.ex

```diff
@@ -63,7 +63,7 @@ defmodule CogApi.HTTP.Old do
         %{"id" => id} ->
           {:ok, id}
         nil ->
-          {:error, %{"error" => "Resource not found"}}
+          {:error, %{"errors" => "Resource not found"}}
```

Comment:
Should we also decide to consistently return a list from there so that we can handle multiple errors?